docs/installation.md

```diff
@@ -175,6 +175,14 @@ sudo yum module install -y container-tools:rhel8
 
 The `container-tools:rhel8` is the fast application stream, containing most recent rolling versions of the tools. Use the `container-tools:2.0` stream for stable versions of Podman 1.6. The command `yum module list container-tools` shows the available streams.
 
+#### [RHEL9](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9)
+
+To install Podman, use the following command in RHEL9:
+
+```bash
+sudo dnf install podman
+```
+
```

Comment:
@iamgini Thanks for the update.  However, Podman should be installed by default starting in a least RHEL 8, and I think RHEL 7.  Is this needed?  If you think it's helpful from an update perspective, I'm good with adding it.

Also, it looks like you didn't sign your commit and the CI system will not let us merge it until it is signed `git commit --amend -s`, rebase, push --force.